addons/im_livechat/static/src/embed/cors/boot.js

```diff
@@ -23,7 +23,7 @@ import { session } from "@web/session";
         {
             async: true,
             start(env) {
-                return function rpc(route, params = {}, settings) {
+                return function _rpc(route, params = {}, settings) {
```

Comment:
That won't work: it's a patch of the rpc service in a specific context.

Reply:
oups indeed, I'll fix it